src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -21,7 +21,7 @@ ferc_to_sqlite_settings:
   ferc60_xbrl_to_sqlite_settings:
     years: [2021, 2022]
   ferc714_xbrl_to_sqlite_settings:
-    years: [2021, 2022]
+    years: [2021, 2023]
```

Comment:
the years here and in the pudl-etl settings were disjoint (here had 2022 and the pudl etl settings had 2023. this was causing several errors and was hard to track down bc it only happened when i re-ran the xbrl_to_sqlite stuff. because of this i added a validation (and corresponding unit test) in to the EtlSettings